File: assignment/module 03/5)string/Q14.cpp
//14.Write a program in C to combine two strings manually..

#include<stdio.h>

main()

{
	char a[30],b[30];
	int i,j;
	
	printf("Enter A string:");
	gets(a);
	
	printf("Enter B string:");
	gets(b);                    //taking two strings input
	
	for (i=0;a[i]!='\0';i++);  //loop for first input
	
	
	for (j=0;b[j]!='\0';j++) //loop for second input
	{
		a[i]=b[j];
		i++;
	}
	a[i]='\0';
	{
		printf("after combining two strings  %s  ",a);
	}
	
	
}

File: Python/Assignment/Module-2 (Fundamentals of python/Q14.py
#Write a Python program to get a single string from two given strings, 
#separated by a space and swap the first two characters of each string.  

s = input("Enter the string:")
s1= input("Enter the second string:") #user input

if len(s)<2 or len(s1)<2:
    print("both string atleast two characters") #strings atleast have more than two characters
else:
    new_s = s[:2]+ s1[2:]  #swapping first character of string
    new_s1= s1[:2]+ s[2:]

print(new_s,new_s1)        